doc/source/whatsnew/v2.3.3.rst

```diff
@@ -8,6 +8,17 @@ including other versions of pandas.
 
 {{ header }}
 
+.. _whatsnew_220.py14_compat:
```

Comment:
This looks like a misformatted anchor (C&P artefact)? C.f. `_whatsnew_233` on the next item:
https://github.com/pandas-dev/pandas/blob/179258fb8048999ba4cad675a7608f09fff88579/doc/source/whatsnew/v2.3.3.rst?plain=1#L23

Still present on main
https://github.com/pandas-dev/pandas/blob/179258fb8048999ba4cad675a7608f09fff88579/doc/source/whatsnew/v2.3.3.rst?plain=1#L11